packages/x-components/src/x-modules/queries-preview/wiring.ts

```diff
@@ -59,6 +67,9 @@ export const queriesPreviewWiring = createWiring({
   QueryPreviewUnmountedHook: {
     clearQueryPreviewWire
   },
+  QueriesPreviewDestroyedHook: {
```

Comment:
The event `QueryPreviewUnmountedHook` is already there to do the same that `QueriesPreviewDestroyedHook`